python/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
```

Comment:
please use PyPI as the preferred source for Python packages. That also resolves the issue you have in the current Portfile with the `version`; the `v` should not be part of it.

Reply:
@reneeotten There is no source archive on PyPI. I already opened an issue for this: https://github.com/rsmenon/pygments-mathematica/issues/26

https://pypi.org/project/pygments-mathematica/#files

I assumed that it is either not possible (or not preferred) to use the .whl file, and the only other option I could find was to get the sources from GitHub. Please let me know how to proceed.

Reply:
@reneeotten The `v`, as well as the incorrect Portfile location, are now fixed.

Reply:
oh, sorry I missed there was only a `.whl` file - you're right that's certainly not preferred!